fcl/dunefd/g4/standard_g4_dune10kt.fcl

```diff
@@ -39,13 +38,12 @@ physics:
      largeant:    @local::dunefd_larg4
      IonAndScint: @local::dunefd_ionandscint
      PDFastSim:   @local::dunefd_pdfastsim
-     elecDrift:   @local::dunefd_elecdrift
```

Comment:
I understand the point here but we do use the elecDrift module to get the correct deposited energy in the FlashMatch analyzer. The migration of using elecDrift to wirecell was put on hold because of differences in the output value between the two. It may be because of energy deposited outside the drift window or it can be because of the energy deposited in the CRP gaps (sorry, I think this has been checked for VD only yet). @absolution1  Could we hold this change until we sort this out?

Reply:
If its still in use then it should stay for now.  @mhlswang was there a particular reason for removal (I would guess space/memory savings)?

Reply:
Hi @mhlswang how much is the PR/work/studies impacted if elecdrift is kept?

Reply:
Ok, I have made a few tests and it seems everything works fine with replacing ElectDrift by wirecell in HD in the flashmatch analyzer. So we can go ahead with this change and work out the problem in VD afterwards. I will make the necessary changes in duneopdet.

Reply:
@absolution1, it isn't a big deal to keep elecDrift.  Only reason to take it out was to try to reduce the output files by removing something we weren't going to use.  But I see that @lpaulucc has just verified that wirecell works fine too for flashmatch.